docs/developers/contributing.rst

```diff
@@ -245,9 +211,7 @@ notes (``docs/release.rst``).
 
 The documentation can be built locally by running::
 
-    $ cd docs
-    $ make clean; make html
-    $ open _build/html/index.html
+    $ hatch --env docs run build
```

Comment:
Hmm, this doesn't work for me:

```
% hatch --env docs run build
sphinx-build -b html -d _build/doctrees  -W --keep-going . _build/html
/Users/dstansby/Library/Application Support/hatch/env/virtual/zarr/4XDzdpBV/docs/bin/sphinx-build: line 2: /Users/dstansby/Library/Application Support/hatch/env/virtual/zarr/4XDzdpBV/docs/bin/python: No such file or directory
/Users/dstansby/Library/Application Support/hatch/env/virtual/zarr/4XDzdpBV/docs/bin/sphinx-build: line 2: exec: /Users/dstansby/Library/Application Support/hatch/env/virtual/zarr/4XDzdpBV/docs/bin/python: cannot execute: No such file or directory
make: *** [html] Error 126
```

Would be good to either:
- Fix in this PR
- Change back to the old instructions (which at least seem to work for me with the right deps installed) and open an issue to update this later